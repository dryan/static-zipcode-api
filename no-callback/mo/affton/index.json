{"region": {"fips": "29", "name": "Missouri", "abbr": "MO"}, "postal_codes": ["63123"], "name": "Affton", "counties": [{"fips": "189", "name": "St Louis"}]}