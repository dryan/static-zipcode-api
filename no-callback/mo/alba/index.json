{"region": {"fips": "29", "name": "Missouri", "abbr": "MO"}, "postal_codes": ["64830"], "name": "Alba", "counties": [{"fips": "097", "name": "Jasper"}]}