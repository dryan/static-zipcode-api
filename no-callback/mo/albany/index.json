{"region": {"fips": "29", "name": "Missouri", "abbr": "MO"}, "postal_codes": ["64402"], "name": "Albany", "counties": [{"fips": "075", "name": "Gentry"}]}