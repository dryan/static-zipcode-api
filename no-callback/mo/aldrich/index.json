{"region": {"fips": "29", "name": "Missouri", "abbr": "MO"}, "postal_codes": ["65601"], "name": "Aldrich", "counties": [{"fips": "167", "name": "Polk"}, {"fips": "039", "name": "Cedar"}, {"fips": "057", "name": "Dade"}]}