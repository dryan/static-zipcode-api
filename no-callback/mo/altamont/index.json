{"region": {"fips": "29", "name": "Missouri", "abbr": "MO"}, "postal_codes": ["64620"], "name": "Altamont", "counties": [{"fips": "061", "name": "Daviess"}]}