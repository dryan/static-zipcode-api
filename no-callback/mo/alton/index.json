{"region": {"fips": "29", "name": "Missouri", "abbr": "MO"}, "postal_codes": ["65606"], "name": "Alton", "counties": [{"fips": "149", "name": "Oregon"}]}