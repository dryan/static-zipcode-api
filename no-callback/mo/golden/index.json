{"region": {"fips": "29", "name": "Missouri", "abbr": "MO"}, "postal_codes": ["65658"], "name": "Golden", "counties": [{"fips": "009", "name": "Barry"}]}